common/arg.cpp

```diff
@@ -2324,5 +2324,47 @@ common_params_context common_params_parser_init(common_params & params, llama_ex
         }
     ).set_examples({LLAMA_EXAMPLE_TTS}));
 
+    add_opt(common_arg(
+        {"--embd-bge-small-en-default"},
+        string_format("use default bge-small-en-v1.5 model (note: can download weights from the internet)"),
+        [](common_params & params) {
+            params.hf_repo = "CompendiumLabs/bge-small-en-v1.5-gguf";
+            params.hf_file = "bge-small-en-v1.5-q4_k_m.gguf";
+            params.pooling_type = LLAMA_POOLING_TYPE_NONE;
+            params.embd_normalize = 2;
+            params.n_ctx = 512;
+            params.verbose_prompt = true;
+            params.embedding = true;
+        }
+    ).set_examples({LLAMA_EXAMPLE_EMBEDDING, LLAMA_EXAMPLE_SERVER}));
+
+    add_opt(common_arg(
+        {"--embd-e5-small-en-default"},
+        string_format("use default e5-small-v2 model (note: can download weights from the internet)"),
+        [](common_params & params) {
+            params.hf_repo = "ChristianAzinn/e5-small-v2-gguf";
+            params.hf_file = "e5-small-v2.Q4_K_M.gguf";
+            params.pooling_type = LLAMA_POOLING_TYPE_NONE;
+            params.embd_normalize = 2;
+            params.n_ctx = 512;
+            params.verbose_prompt = true;
+            params.embedding = true;
+        }
+    ).set_examples({LLAMA_EXAMPLE_EMBEDDING, LLAMA_EXAMPLE_SERVER}));
+
+    add_opt(common_arg(
+        {"--embd-gte-small-default"},
+        string_format("use default gte-small model (note: can download weights from the internet)"),
+        [](common_params & params) {
+            params.hf_repo = "ChristianAzinn/gte-small-gguf";
```

Comment:
Ideally, we should curate a set of models in the [ggml-org](https://huggingface.co/ggml-org) HF organization account. This would make the configs more future-proof and prevent certain attack vectors where the models are swapped with malformed versions.

Reply:
Does this entail creating a collection in the `ggml-org` account and then pushing these models to it? 
Is it alright if I create a new collection, and would a name like `Embedding Models` be an appropriate name?

Reply:
What I normally do is use the [GGUF My Repo](https://huggingface.co/spaces/ggml-org/gguf-my-repo) space to create the GGUF model from the original source. After that I transfer the repo to the `ggml-org`. This way the trace to the original model is preserved:

<img width="636" alt="image" src="https://github.com/user-attachments/assets/3c73c8be-ca58-4459-adad-b12b6c25c50f" />

Not sure if this is the simplest process to do this, but it works.

The collections can be created at any point - they are simply lists to which you can add and remove models at any time. I think a single, generic collection (e.g. `llama.cpp presets`) for now would be fine. Later we can reorganize if necessary.

Reply:
Thanks for the details about how to do this! I think I've followed [them](https://github.com/danbev/learning-ai/blob/main/notes/llama.cpp/embeddings-presets.md#huggingface-org-curated-models) and transferred one [model](https://huggingface.co/collections/ggml-org/llamacpp-presets-67a471d0f199e648eaa8a01e) as a start. Let me know if this looks alright and I'll do the same thing with the other ones.

Reply:
Yes, this seems correct. There is one annoying detail that the GGUF'ed repo readme keeps the old model id, from before the transfer. So what I normally do is to edit it like this:

https://huggingface.co/ggml-org/Qwen2.5-Coder-0.5B-Q8_0-GGUF/commit/5788aee90e725673490be18f553d234feaa8303d

I wish there was a way to directly make the repo inside the organization and not have to do this extra step.